.gitpod.yml

```diff
@@ -64,34 +64,12 @@ ports:
   # Ignore host https port
   - port: 8443
     onOpen: ignore
-  - port: 3306
```

Comment:
Why remove 3306? It's the port the database is bound to.

Reply:
I remove `3306` because I don't see it used in Gitpod panel.

Also - you can open this PR, and see that there is no warning or message about `3306`

Reply:
restored